src/modes/play/objects/growthFungus.js

```diff
@@ -0,0 +1,63 @@
+import Mob from './mob';
+
+class GrowthFungus extends Mob {
+  constructor(game, mode, x, y, behavior) {
+    super(game, mode, x, y);
+    this.name = "GrowthFungus";
+    this.color = "#e72";
+    this.brain = new GrowthFungusBrain(this, behavior);
+    this.spriteBank = {
+      standing: ['assets/sprites/mobs/fungus/fungus-1.png'],
+      running: ['assets/sprites/mobs/fungus/fungus-1.png', 'assets/sprites/mobs/fungus/fungus-2.png'],
+    }
+  }
+
+  get animationRate() {
+    return Math.random() * 10000;
+  }
+
+  set animationRate(val) {
+    return 'lol';
+  }
+
+  collisionCallback(obj) {
```

Comment:
For the future: Collision callbacks need to be rock solid at handling both idempotent collision events (e.g. "I collide with a cookie, my cookie counter increments by exactly one regardless of how many times the event fires") and continuous collision events (e.g. "As long as I'm touching this surface, my friction is reduced to 0.").

Right now, those situations are being handled kind of wherever... But it would be nice to handle that right up here.